po/fr.po

```diff
@@ -176,11 +176,11 @@ msgstr "Mot de passe de protection"
 
 #: TwoFactorAuth/widgets/settings.py:93
 msgid "Auto-lock the application (m):"
-msgstr "Auto-verrouillage de l'application (m) :"
+msgstr "Auto-verrouillage de l'application (m):"
 
 #: TwoFactorAuth/widgets/settings.py:112
 msgid "Secret code generation time (s) :"
```

Comment:
This extra whitespace looks like a problem in the source message.  It is inconsistent with the other messages.

@bil-elmoussaoui, should the source string be fixed to be `"Secret code generation time (s):"` ?